src/line_launcher/mod.rs

```diff
@@ -16,17 +19,17 @@ const VELOCITY: u8 = 100;
 pub struct LineLauncher {
     beat_message_receiver: Receiver<BeatNumber>,
     state: State,
-    notes: Vec<u8>,
-    output: MidiOutputConnection,
+    lines: Vec<Line>,
```

Comment:
Not using the functionality of this being more than one line yet, but sets it up for eg being able to choose different lines at random